tests/test_workers/test_tasks/test_build.py

```diff
@@ -11,7 +11,8 @@
 import ruamel.yaml
 
 from iib.exceptions import IIBError
-from iib.workers.tasks import build
+from iib.workers.tasks import build, utils
+from iib.workers.tasks.utils import RequestConfigAddRm, get_binary_image_from_config, verify_labels
```

Comment:
THis is a modification of the previous line

Reply:
thanks for catching this one.